tests/phpunit/bootstrap.php

```diff
@@ -9,6 +9,8 @@
  */
 
 define( 'TESTS_PLUGIN_DIR', dirname( dirname( __DIR__ ) ) );
+// Ensure Composer autoloader is available.
+require_once TESTS_PLUGIN_DIR . '/vendor/autoload.php';
```

Comment:
Moving this here seems a bit unrelated, was there a specific problem that led you to doing that? In any case, I agree having that here outside of the condition below makes more sense.

Reply:
It's now needed in all cases (not only when using wp-phpunit locally) since we use it to load the PHPUnit Polyfills library, but also the added `packages/wp-phpunit-polyfill` package which is loaded by Composer.

Reply:
Turns out this actually isn't needed at all because PHPUnit loads it automatically 😄